Core/Core/People/User.swift

```diff
@@ -53,6 +53,15 @@ extension User: WriteableModel {
 }
 
 extension User {
+    public static func displayName(_ name: String, pronouns: String?) -> String {
+        return [
+            name,
+            pronouns.flatMap { "(\($0))" },
+        ]
+        .compactMap { $0 }
+        .joined(separator: " ")
+    }
```

Comment:
I couldn't decide where to put `personDisplayName(...)` like we talked about so I settled on `User.displayName(...)`. Let me know thoughts.

Reply:
This name is great. I was wondering if we were going to do an NSAttributedString so that the pronouns could be italic or a little different.